test/Integration/IntegrationTestBase.cs

```diff
@@ -324,17 +342,6 @@ public void Dispose()
             {
                 this.TestOutput.WriteLine($"Failed to close connection. Error: {e1.Message}");
             }
-            try
```

Comment:
Why are you changing these?

Reply:
Changed the order for function host to be killed before the database is dropped. As the trigger periodically checks for changes in the database, the database was not able to be dropped previously.